src/index.html

```diff
@@ -1,13 +1,52 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="conteiner">
+      <div class="Product">
+        <div class="Product__img" href="#">
+          <img src="images/imac.jpeg" 
+               alt="APPLE A1419 iMac 27 "
+               value="APPLE A1419 iMac 27 Retina 5K (MNED2UA)"
+          >
```

Comment:
```suggestion
          <img 
            src="images/imac.jpeg" 
            alt="APPLE A1419 iMac 27 "
            value="APPLE A1419 iMac 27 Retina 5K (MNED2UA)"
          >
```

Reply:
move attributes to new line when there are more than 2 attrs. If you started it - move every attr with +2 spaces indentation